test/unit/org/apache/cassandra/schema/CreateTableValidationTest.java

```diff
@@ -81,7 +80,10 @@ public void testCreateKeyspaceTableWarning() throws IOException
             DatabaseDescriptor.setKeyspaceCountWarnThreshold(Schema.instance.getKeyspaces().size() + 1);
             query = new QueryMessage(String.format(createKeyspace, 2), QueryOptions.DEFAULT);
             resp = client.execute(query);
-            assertTrue(resp.getWarnings() == null || resp.getWarnings().isEmpty());
+            List<String> warns = resp.getWarnings();
+            if (warns != null)
+                warns.removeIf(s -> s.contains("is higher than the number of nodes"));
```

Comment:
I understand that we still have warnings here despite of having a RF=1 because of the RF of `system_distributed` and `system_traces`. Indeed, when we run a single instance we always have warnings for those in the logs. I'm wondering whether it would make sense to not emit the warning for system keyspaces.